test/credentials_test.dart

```diff
@@ -299,4 +302,172 @@ Future<void> main([List<String>? args]) async {
       await authProvider.callResetPasswordFunction(username, newPassword);
     }, throws<RealmException>("failed to reset password for user $username"));
   }, appName: AppNames.autoConfirm);
+
+  baasTest('JWT validation by specified public key  - login with new user', (configuration) async {
+    final app = App(configuration);
+    var newUserId = ObjectId();
+    String username = "${generateRandomString(5)}@realm.io";
+    final jwt = JWT(
+      {
+        "sub": "$newUserId",
+        "name": {"firstName": "John", "lastName": "Doe"},
+        "email": username,
+        "gender": "male",
+        "birthDay": "1999-10-11",
+        "minAge": "10",
+        "maxAge": "90",
+        "company": "Realm",
+      },
+      issuer: 'https://realm.io',
+      audience: Audience(["mongodb.com"]),
+    );
+
+    String privateKey = File("test/data/jwt_keys/private_key.pem").readAsStringSync();
```

Comment:
Consider parsing this as an argument to the test exe to allow to run on Android and iOS as well

Reply:
It is no more needed. We don't use private key anymore.